DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/DelegateGroupsControllerTests.cs

```diff
@@ -35,6 +37,7 @@ public class DelegateGroupsControllerTests
         private IGroupsService groupsService = null!;
         private HttpRequest httpRequest = null!;
         private HttpResponse httpResponse = null!;
+        private IJobGroupsDataService jobGroupsDataService = null!;

```

Comment:
This and some other unused private services can be removed.

Reply:
removed